services/brig/test/integration/API/User/Auth.hs

```diff
@@ -361,6 +368,28 @@ testSendLoginCode brig = do
   let _timeout = fromLoginCodeTimeout <$> responseJsonMaybe rsp2
   liftIO $ assertEqual "timeout" (Just (Code.Timeout 600)) _timeout
 
+testLoginVerify6DigitEmailCode :: Brig -> Galley -> DB.ClientState -> Http ()
+testLoginVerify6DigitEmailCode brig galley db = do
```

Comment:
TODO add tests for: 
* testing login fails with a wrong code
* testing login fails with an expired code